2011-11-03  Yavor Doganov  <yavor@gnu.org>

	* configure.ac: Force GS_FAKE_MAIN on GNU/kFreeBSD as programs
          with more than 512 args abort (workaround for #644868).


--- gnustep-base.orig/configure.ac
+++ gnustep-base/configure.ac
@@ -2346,6 +2346,7 @@
   case "$target_os" in
     freebsd2*)	enable_fake_main=yes;;
     freebsd*)	;;
+    kfreebsd*)	enable_fake_main=yes;;
     netbsd*)	enable_fake_main=yes;;
     openbsd*)	enable_fake_main=yes;;
     *sysv*)	enable_fake_main=yes;;
